CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### MultiQC updates
 
+- Fixed fastp `%PF` ([#1574](https://github.com/ewels/MultiQC/issues/1574))
```

Comment:
Right idea but wrong place. Should be further down under `### Module updates` with a `- Fastp` bullet and then an indented bullet for the issue fixed..

Reply:
Sorry!!!

Reply:
No problem!